apps/src/constants.js

```diff
@@ -57,6 +57,7 @@ exports.TestResults = {
   LOCAL_FUNCTION_FAIL: -6,            // The program contains an unexpected JavaScript local function
   GENERIC_LINT_FAIL: -7,              // The program contains a lint error
   LOG_CONDITION_FAIL: -8,             // The program execution log did not pass a required condition
+  SKIPPED: -9,                        // Skipped, e.g. they used the skip button on a challenge level
```

Comment:
Yeah I'll split up the dashboard and apps parts and merge the dashboard part first. If you skip and hit an old frontend during the deploy, the starter code will be missing if you come back to the puzzle later.

Saved but unsubmitted progress should take precedence over a skip, I made it `-100` instead.